mtags/src/main/scala/scala/meta/internal/pc/AutoImports.scala

```diff
@@ -48,18 +54,20 @@ trait AutoImports { this: MetalsGlobal =>
           case Some(pkg)
               if pkg.symbol != rootMirror.EmptyPackage ||
                 pkg.stats.headOption.exists(_.isInstanceOf[Import]) =>
-            val lastImport = pkg.stats
+            val lastImport: (Tree, Boolean) = pkg.stats
               .takeWhile(_.isInstanceOf[Import])
-              .lastOption
-              .getOrElse(pkg.pid)
+              .lastOption // if there are no imports, return true to pad top of import
+              .fold[(Tree, Boolean)]((pkg.pid, true))((tree => (tree, false)))
```

Comment:
is the explicit type annotation here necessary?

Reply:
Without it I get a type mismatch where it wants a RefTree on the right where it's just a Tree.

Reply:
ah, I would've assumed it were unnecessary since the entire expression is annotated anyway. Too bad